index.js

```diff
@@ -195,6 +195,11 @@ const Synopsis = function (options) {
         delete entry.score
       }
     }
+
+    if (entry.window == null) {
```

Comment:
to be consistent, can we do this on one-line, i.e.,

              if (!entry.window) entry.window = []

thanks!

Reply:
done

Reply:
if you wanted to get fancy:
`entry.window = entry.window || []` 😛